feedback_topics: 
    base_cyl: /roboclaw_driver/128/motor1/joint_state
    cyl_arm1: /roboclaw_driver/129/motor1/joint_state
    arm1_arm2: /roboclaw_driver/129/motor2/joint_state
    arm2_arm3: /roboclaw_driver/130/motor1/joint_state
    arm3_tool: /roboclaw_driver/130/motor2/joint_state

command_topics:
    base_cyl: /roboclaw_driver/128/motor1/set_joint_state
    cyl_arm1: /roboclaw_driver/129/motor1/set_joint_state
    arm1_arm2: /roboclaw_driver/129/motor2/set_joint_state
    arm2_arm3: /roboclaw_driver/130/motor1/set_joint_state
    arm3_tool: /roboclaw_driver/130/motor2/set_joint_state

offset_base_cyl: 0.0
offset_cyl_arm1: -0.7853981633974483
offset_arm1_arm2: 2.356194490192345
offset_arm2_arm3: 0.5
offset_arm3_tool: 0.0